Add a new English video (#1811)
pingcap/docs-tidb-operator: en/get-started.md

5. [Upgrade a TiDB cluster](#step-5-upgrade-a-tidb-cluster)
6. [Destroy the TiDB cluster and the Kubernetes cluster](#step-6-destroy-the-tidb-cluster-and-the-kubernetes-cluster)

> You can watch the below video (for about 12 minutes) that shows the process of getting started with TiDB Operator.

Author: ```suggestion
You can watch the following video (about 12 minutes) that shows the process of getting started with TiDB Operator.
```

> You can watch the following video (about 12 minutes) that shows the process of getting started with TiDB Operator.

Reviewer: ```suggestion
You can watch the following video (about 12 minutes) to learn how to get started with TiDB Operator.
```
